src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -155,20 +157,120 @@ public void cannotUpdateCirculationRulesWithOverdueFinePolicyId() {
   public void cannotUpdateCirculationRulesWithLostItemPolicyId() {
 
     String rule = circulationRulesFixture.soleFallbackPolicyRule(
-      loanPoliciesFixture.canCirculateFixed().getId().toString(),
-      requestPoliciesFixture.allowAllRequestPolicy().getId().toString(),
-      noticePoliciesFixture.activeNotice().getId().toString(),
-      overdueFinePoliciesFixture.facultyStandard().getId().toString(),
-      UUID.randomUUID().toString());
+      loanPoliciesFixture.canCirculateFixed().getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.activeNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandard().getId(),
+      UUID.randomUUID());
 
-    Response response = circulationRulesFixture
-      .attemptUpdateCirculationRules(rule, "i");
+    Response response = circulationRulesFixture.attemptUpdateCirculationRules(rule);
 
     assertThat(response.getStatusCode(), is(422));
     assertThat(response.getJson().getString("message"),
       is("The policy i does not exist"));
   }
 
+  @Test
+  public void canUpdateCirculationRulesWithTwentyExistingLoanPolicies() {
+
+    Set<UUID> loanPolicyIds = getSetOfPolicyIds(20);
+    addPoliciesToLoanPolicyFixture(loanPolicyIds);
+
+    loanPolicyIds.forEach(loanPolicyId -> {
+      String rule = circulationRulesFixture.soleFallbackPolicyRule(
+        loanPolicyId, requestPoliciesFixture.allowAllRequestPolicy().getId(),
+        noticePoliciesFixture.activeNotice().getId(),
+        overdueFinePoliciesFixture.facultyStandard().getId(),
+        lostItemFeePoliciesFixture.facultyStandard().getId());
+      Response response = circulationRulesFixture.attemptUpdateCirculationRules(rule);
+      assertThat(String.format(
```

Comment:
```suggestion
        lostItemFeePoliciesFixture.facultyStandard().getId());
        
      Response response = circulationRulesFixture.updateCirculationRules(rule);
      
      assertThat(String.format(
```

`attempt*` methods usually used when we expect a failure response, since we expect success here, I'd use the `updateCirculationRules` instead.

Reply:
We already have updateCirculationRules method, but it has own assert and void return type.
It is used from many places. Using it causes 4 or 5 sonar smell issues. This is a reason why I had to use attempt* instead of update* method.

Do you think it makes sense to create one more update method with the same code as attempt*? Or just call attempt* inside the update*?

Reply:
Aha, got it. Let's leave it as is then. Just in case `update*` will be accidentally modified and don't assert response code.